client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,5 @@
+export const TAXRATES_REQUEST = 'TAXRATES_REQUEST';
```

Comment:
I feel like these should still be prefixed with `WOOCOMMERCE_`, as the reducers and actions are all under `woocommerce/state`.